lib/team.rb

```diff
@@ -0,0 +1,19 @@
+class Team
+  attr_accessor :country,
```

Comment:
Try to make sure you're only exposing the methods that you actually want your team members to use in the future. Here the requirement is that you allow `eliminated` to be set from the outside. Would prefer that `country` and `players` remain as `attr_reader`s.

Additionally, the spec says that the method used to access the state of `eliminated` is `eliminated?`, with the question mark implying that we're returning a boolean. Would switch the `attr_accessor` for `eliminated` to an `attr_writer` and then create a method `eliminated?` to allow people to read that value:

```ruby
def eliminated?
  @eliminated
end
```

I know it seems nit picky, but you're telling a story with your code, and that question mark is part of the story. Telling users of your class that they can't simply reach in and change the value of `country` or `players` once they've been set is part of the story. They're instructions on how to use the class.

Encapsulation is one of the four pillars of OOP, and using it as a guiding principle means (to me at least) that you don't expose more functionality to the users of the class than you need/want to. Allowing outside users to access `country` and `players` directly may lead them to use your class in ways you didn't expect. Once you've done that, then there's a decent chance that you have to continue supporting that behavior (perhaps with added checks to see that people aren't abusing that access) even if you'd rather not.